Kconfig

```diff
@@ -41,21 +41,10 @@ config IMAGE_BOOT_PG1
 	  allows to recover. Otherwise you risk to BRICK THE IOT2050!
 
 config IMAGE_BOOT_PG2
-	bool "Firmware image for PG2 devices"
```

Comment:
should we also add this explanation to the README?

Reply:
It's in the Kconfig description, and the README points there already.